tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 envlist =
-    py{36,37,38,39}-twisted{187,latest}
+    py{36,37,38,39}-twisted{187,latest}-{ubuntu,macos}
+    py{36,37}-twistedlatest-windows
```

Comment:
why the limit on twisted versions on windows? Also, might it be worth increasing the minimum twisted version first - 18.7 was released june 2018, and 20.3 is even more than 6 months old.

Reply:
The tests fail if there's no wheel.  Which is the case for [18.7.0](https://pypi.org/project/Twisted/18.7.0/#files) and Python > 3.7.

Reply:
Ah fair. Sounds like a reason to bump the minimum supported version 😁

Reply:
FYI: Twisted 19.7.0 is the first version that provides wheels for Python 3.6 and 3.7 on Windows: https://pypi.org/project/Twisted/19.7.0/#files